build.rs

```diff
@@ -770,7 +770,7 @@ fn configure(interpreter_config: &InterpreterConfig) -> Result<String> {
     }
 
     if let Some(minor) = interpreter_config.version.minor {
-        for i in 5..(minor + 1) {
+        for i in 6..(minor + 1) {
```

Comment:
:slightly_smiling_face:

Reply:
you could have used `6..=minor` instead of `6..(minor + 1)`

Reply:
:+1: good idea!